lib/assert.js

```diff
@@ -60,11 +60,21 @@ function isSWEnv() {
     self instanceof ServiceWorkerGlobalScope);
 }
 
+function isValue(object, expectedValue) {
+  const parameter = Object.keys(object).pop();
```

Comment:
If `object` were a different type (e.g `Object.keys([]).pop()`), this would result in `actualValue` = object[undefined] if we weren't doing type validation before consuming the `object`. 

It might be worth checking your type  before using it. That said, there's prior art in this file as we aren't checking this in the `isType` function either. Your call on whether to change the pattern :)